README.md

```diff
@@ -1,2 +1,57 @@
+![GitHub forks](https://img.shields.io/github/forks/beyond-io/picATrip?style=social) ![GitHub issues](https://img.shields.io/github/issues-raw/beyond-io/picATrip?logo=GitHub) ![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/beyond-io/picATrip?logo=GitHub) ![GitHub top language](https://img.shields.io/github/languages/top/beyond-io/picATrip)
 # picATrip
-picATrip - Beyond 05, team 2 project
+![logo- picATrip](https://user-images.githubusercontent.com/48455448/110465508-26ae0a80-80dd-11eb-8268-586b26a1e59e.png)
+
+### **Who are we?**
+
+   picATrip is A web platform for **sharing experiences from _trips_** in a form of **photos** based
+   on various parameters as
+   location/duration time/difficulty and provides a trip **search functionality** and **commenting
+   functionality**.
+
+   It allows users to have the opportunity to **contribute their trip experience** and thoughts by
+   **pic**king  a picture of a **trip** shared on the platform,  and by that **learn about the trips**
+   across the country that stands behind the pictures and having the opportunity to **share their
+   thoughts** and **comment** if they desire.
+
+### Why using us?
+  **Ever heard of the phrase 'eating with the eyes'?**, well that's about it! we believe in outdoor lifestyle and trips, and we strive
```

Comment:
Maybe start the sentence like "Have you ever heard..." ? 
Also I think you should remove the comma after the question mark.

Reply:
Can you delete the comma after the question mark as well ?